src/patternfly/components/Tile/tile.scss

```diff
@@ -99,15 +93,12 @@
 
   &.pf-m-selected,
   &:focus {
-    &::before {
-      --pf-c-tile--before--BorderWidth: var(--pf-c-tile--before--m-selected--BorderWidth);
-      --pf-c-tile--before--BorderColor: var(--pf-c-tile--before--m-selected--BorderColor);
-    }
-
-    cursor: pointer;
-
     --pf-c-tile__title--Color: var(--pf-c-tile--m-selected__title--Color);
     --pf-c-tile__icon--Color: var(--pf-c-tile--m-selected__icon--Color);
+    --pf-c-tile--before--BorderWidth: var(--pf-c-tile--m-selected--before--BorderWidth);
+    --pf-c-tile--before--BorderColor: var(--pf-c-tile--m-selected--before--BorderColor);
+
+    cursor: pointer;
```

Comment:
could `cursor: pointer` just be set on the entire component? The card sets `.pf-m-selectable { cursor: pointer; }`